unit-tests.log

```diff
@@ -0,0 +1 @@
+test
```

Comment:
necessary?

Reply:
I figured it was based on this PR in wire-web-package https://github.com/wireapp/wire-web-packages/pull/6205/files#diff-7739ac093c597ada5e78f6e0fc4fcb011497e85cff3d5e1e8cd45857f3c962d2R1